src/main/java/com/faforever/api/config/elide/ElideConfig.java

```diff
@@ -130,4 +134,16 @@ public EntityDictionary entityDictionary() {
 
     return new EntityDictionary(checks);
   }
+
+  @Bean
+  public Elide initElideSearch(SpringHibernateDataStore springHibernateDataStore) {
```

Comment:
I don't think you should create another factory method for Elide class. There is one already in the beginning of the file